### ISSUE TITLE
Add extension.json file

### ISSUE THREAD

**Issue by wenzhixin** [prograhammer/bootstrap-table-contextmenu#1]
Hi, can you add an `extension.json` file to your repo. 

Like this: [extension.json](https://github.com/wenzhixin/bootstrap-table/blob/develop/src/extensions/editable/extension.json), and then the extension will show in this page: http://bootstrap-table.wenzhixin.net.cn/extensions/

**Comment by prograhammer** [comment]
Hi @wenzhixin 

I added a [demo](http://www.prograhammer.com/demos/bootstrap-table-contextmenu) link to the readme.md and tagged the repo at version 1.0.0. I also added the [exention.json](https://github.com/prograhammer/bootstrap-table-contextmenu/blob/master/extension.json) file. But did I do it right? I don't see in on the link you provided. And what did you want for the "url" and "example" sections in the file?
